metrics/test_driver/cli/flutter/runner/flutter_drive_process_runner.dart

```diff
@@ -0,0 +1,54 @@
+import '../../../common/config/browser_name.dart';
+import '../../../common/config/device.dart';
+import '../../common/runner/process_runner.dart';
+import '../command/drive_command.dart';
+import '../command/flutter_command.dart';
+import '../process/flutter_process.dart';
+
+/// Runs the flutter driver tests in separate process.
```

Comment:
Suggestion to indicate that it runs in separate process in FlutterProcess class document or drop it here to make sure that description matches. I guess the question here is it understandable that every runner, runs in the separate process?